lib/data.dart

```diff
@@ -62,6 +62,55 @@ class StudentFeedback {
       this.uid, this.email, this.upvotes, this.downvotes);
 }
 
+class FeedbackComment {
+  String commentId;
+  // String feedbackId;
+  String uid;
+  String email;
+  DateTime date;
+  String text;
+
+  FeedbackComment(
+      this.commentId,
+      // this.feedbackId,
+      this.uid, this.email, this.date, this.text);
+}
+
+Stream<List<FeedbackComment>> getComments(String feedbackId) {
```

Comment:
A stream is unnecessary, but ok.

Reply:
its for quick updating (copied your solution with feedbacks btw)